build.gradle

```diff
@@ -20,46 +20,20 @@ buildscript {
     maven { url "https://plugins.gradle.org/m2/" }
   }
   dependencies {
-    classpath "com.netflix.spinnaker.gradle:spinnaker-dev-plugin:5.2.1"
-    classpath "org.junit.platform:junit-platform-gradle-plugin:1.0.0"
-    classpath "com.netflix.nebula:nebula-kotlin-plugin:$kotlinVersion"
-    classpath "org.jetbrains.dokka:dokka-gradle-plugin:0.9.15"
+    classpath "com.netflix.spinnaker.gradle:spinnaker-dev-plugin:5.2.2"
   }
 }
 
+plugins {
+  id("nebula.kotlin") version "1.3.21" apply false
+  id("org.jetbrains.dokka") version "0.9.18" apply false
+}
+
 allprojects {
   apply plugin: "spinnaker.base-project"
-  apply plugin: "java"
 
   group = "com.netflix.spinnaker.swabbie"
 
-  ext {
-    spinnakerDependenciesVersion = '1.31.0'
-    if (project.hasProperty('spinnakerDependenciesVersion')) {
-      spinnakerDependenciesVersion = project.property('spinnakerDependenciesVersion')
-    }
-  }
-
-  def checkLocalVersions = [spinnakerDependenciesVersion: spinnakerDependenciesVersion]
-  if (ext.has('versions')) {
-    def extVers = ext.get('versions')
-    if (extVers instanceof Map) {
-      checkLocalVersions.putAll(extVers)
-    }
-  }
-
-  def localVersions = checkLocalVersions.findAll { it.value.endsWith('-SNAPSHOT') }
-  if (localVersions) {
-    logger.info("Enabling mavenLocal repo for $localVersions")
-    repositories {
-      mavenLocal()
-    }
```

Comment:
wont we need something like this if kork is published locally?

Reply:
Yeah, that's true. I'll give it some thought. Right now I can't actually publish kork locally because of fucking Javadoc so I'll probably try to fix that first.

Reply:
Basically you'd want to enable `mavenLocal()` if the `kork-bom` version ends in `-SNAPSHOT`. Should be pretty easy.